pkg/cmd/config/config.go

```diff
@@ -0,0 +1,149 @@
+package config
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"path/filepath"
+	"reflect"
+	"strings"
+)
+
+type Config struct {
+	LogLevel  string `json:"log_level"`
+	Progress  string `json:"progress"`
+	ConfigDir string `json:"config_dir"`
+	Verbose   int    `json:"verbose"`
```

Comment:
Verbose is just an alias for `LogLevel = output.LogLevelDebug`, so I don't know that we need to support it in the config file.

Reply:
sure, removing it from config file in that case